src/PhpSpreadsheet/Writer/Html.php

```diff
@@ -596,6 +596,12 @@ private function extendRowsForChartsAndImages(\PhpOffice\PhpSpreadsheet\Workshee
         }
         $html = '';
         ++$colMax;
+
+        //Return $html if $rowMax didn't change
+        if ($row !== $rowMax) {
+            return $html;
```

Comment:
Okay, I then should not mark so quickly for approval. Better reviewing first. :-)